pkg/metrics/metrics.go

```diff
@@ -56,8 +71,39 @@ func RegisterMetrics() error {
 	return nil
 }
 
-// UpdatePlaceholderGauge ...
-func UpdatePlaceholderGauge() {
+// UpdateAPIMetrics updates all API endpoint metrics ever 5 minutes
+func UpdateAPIMetrics() {
```

Comment:
Is this called anywhere? This will probably need to be run in a go routine

Reply:
/hold

Reply:
Ahh, could you point me to an example?

Reply:
https://github.com/openshift/certman-operator/blob/master/cmd/manager/main.go#L120

Reply:
Updated